_integrations/index.md

```diff
@@ -13,56 +13,119 @@ redirect_from:
 
-If you have not installed any integrations, you'll be prompted to install them from the Integrations interface. Supported integrations are listed in the **Available** window. 
+OpenSearch Integrations can be installed directly from the [default catalog](https://github.com/opensearch-project/opensearch-catalog/blob/main/docs/integrations/Release.md) that is shipped with every OpenSearch release. Additionally, integrations can be manually loaded and updated using the **Dashboards Management** console to reflect the latest versions or new releases.
 
-To add an integration, select the desired prepackaged assets. Currently, OpenSearch Integrations has two flows: Add or Try it. The following example uses the Try it flow:
+### Installing an integration 
+
+If no integration is installed, you are prompted to install one from the **Integrations** interface. The **Available** dialog window lists supported integrations from the default catalog.
+ 
+![]({{site.url}}{{site.baseurl}}/images/integrations/empty-installed-integrations.png)
+
+To add an integration, select the desired prepackaged assets. OpenSearch Integrations offers the following options: **Add** or **Try it**. The following image shows the **Try it** flow:
+![integrations-observability-catalog.png](/images/integrations/integrations-observability-catalog.png)
+
+The **Available** dialog window allows filtering the list of integrations using the **Categories** dropdown menu, as shown in the following image:
+
+![cloud-integrations-filter.png]({{site.url}}{{site.baseurl}}/images/integrations/cloud-integrations-filter.png)
+
+This example shows how you can navigate and explore the [default catalog](https://github.com/opensearch-project/opensearch-catalog/blob/main/docs/integrations/Release.md) bundled with the OpenSearch release and select the appropriate resources for your system.
 
 1. On the **Integrations** page, select **NginX Dashboard**.
+  ![]({{site.url}}{{site.baseurl}}/images/integrations/nginx-integration.png)
+
 2. Select the **Try it** button. _The Try it flow automatically creates a sample index template, adds sample data to the template, and then creates the integration based on that data._
+
 3. View the asset list and select a dashboard asset.
+  ![]({{site.url}}{{site.baseurl}}/images/integrations/nginx-installed-integration-assets.png)
+
 4. Preview the data visualizations and sample data details.  
+  ![]({{site.url}}{{site.baseurl}}/images/integrations/nginx-integration-dashboard.png)
+
+### Loading new integrations
+
+To load a new integration not included in the OpenSearch release, download it from the [catalog repository](https://github.com/opensearch-project/opensearch-catalog/blob/main/docs/integrations/Release.md), navigate to **Dashboards Management**, and import the new integration.
+
+The following tutorial shows how to download and install a new release or updated integration version from the catalog page to the OpenSearch available integration.
```

Comment:
I'm not following the sentence past "a new release...". Please revise for clarity.